polling_stations/templates/fragments/you_dont_need_poll_card.html

```diff
@@ -1,16 +1,27 @@
 {% load i18n %}
-{% if voter_id_pilot %}
+{% if requires_voter_id %}
     <div class="ds-card">
         <div class="ds-card-body">
             <h3>
                 <span aria-hidden="true">🗳</span>️
-                {{ voter_id_pilot.title }}
+                {% trans "You will need to take photo ID to vote at a polling station in this election" %}
             </h3>
-            <p>{{ voter_id_pilot.detail }}</p>
             <p>
-                <a href="{{ voter_id_pilot.url }}">{% trans "More information on voter ID pilots in your area" context "link text" %}</a>
+                {% trans "You do not need your poll card to vote." %}
             </p>
-            <p>{% trans "You must vote at your assigned polling station." %}</p>
+            <p>
+                {% trans "You must vote at your assigned polling station." %}
+            </p>
+
+            {% if territory == 'NI' %}
+                <p><a href="http://www.eoni.org.uk/Vote/Voting-at-a-polling-place" target="_top">
+                    {% trans "Read more about voting in Northern Ireland" %}
+                </a></p>
+            {% else %}
+                <p><a href="https://www.gov.uk/voting-in-the-uk/polling-stations" target="_top">
```

Comment:
I know we're not getting these translated for these elections, but since you're putting the scaffolding in now, is it possible to blank out the links to the welsh language equivalents of the gov uk and electoral commission websites? if not straightforward , then don't worry about it.

Reply:
On our translated version of the postcode page, we don't link to the CY "voting in the UK" page (https://www.gov.uk/voting-in-the-uk/polling-stations) in this section. The text reads "Darllen mwy am bleidleisio ym Mhrydain Fawr" but it always links to the English language version.

The new link added for the EC website on the home page also doesn't go through to the CY version, and it has no Welsh text translation either. 
<img width="881" alt="Screenshot 2023-03-17 at 10 36 23" src="https://user-images.githubusercontent.com/9531063/225881567-30b4ca82-f3ea-40b5-bdfa-ab97e7575f9e.png">

I don't think there's anything to do here, but I hope I haven't misunderstood your comment 🙊

Reply:
So I know we were only linking to the English version, but I think think there's a way to have 'translated links', so when we come to get translations done, we are reminded to also go and search out the CY versions of the links - I was imagining that it'd by annoying to be reading a site in Welsh, then get linked to the English language version of another site that also has a welsh translation. However not a priority for us!

Reply:
Let's ignore this for now.